### PR TITLE
Data Migration Tool Release 2.1.4

### PR DESCRIPTION
**Pull request by victor-v-rad**
### Tasks
- [MAGETWO-62478](https://jira.corp.magento.com/browse/MAGETWO-62478) Update changelog of Data Migration Tool 2.1.3
- [MAGETWO-63106](https://jira.corp.magento.com/browse/MAGETWO-63106) [GitHub] Merge Pull Request 182 from community
- [MAGETWO-63081](https://jira.corp.magento.com/browse/MAGETWO-63081) DMT 2.1.4 Publication
- [MAGETWO-64347](https://jira.corp.magento.com/browse/MAGETWO-64347) Map files for 1.9.3.2 and 1.14.3.2

### Bugs
- [MAGETWO-59233](https://jira.corp.magento.com/browse/MAGETWO-59233) [GitHub] Integrity constraint violation 1062 Duplicate entry #81
- [MAGETWO-62588](https://jira.corp.magento.com/browse/MAGETWO-62588) [GitHub] Error in SalesIncrement Step while Delta mode is running #203
- [MAGETWO-62729](https://jira.corp.magento.com/browse/MAGETWO-62729) [GitHub] Cannot save category containing products linked to root category after migration #7916

### Bamboo CI builds
- [ ] [Migration Tool](https://bamboo.corp.magento.com/browse/M2-MT2-41)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/data-migration-tool?pullRequest=237) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/data-migration-tool?pullRequest=237) before we can accept your contribution.<br/>**1** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: PascalBrouwers<br/>:x: ygrigoriev<br/>:x: victor-v-rad

File: src/Migration/Step/PostProcessing/Model/ProductsInRootCatalog.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    /**
+     * @var string
+     */
+    private $catalogCategoryProductDocument = 'catalog_category_product';
+
+    /**
+     * Id of category tree root
+     */
+    const TREE_ROOT_ID = 1;
```
Comment (balex13) [inline src/Migration/Step/PostProcessing/Model/ProductsInRootCatalog.php]: Why don't use private variable instead?
Isn't better to retrieve this *Id* instead hardcoding

Comment (victor-v-rad) [inline src/Migration/Step/PostProcessing/Model/ProductsInRootCatalog.php]: Done

**Comment by balex13** [comment]
Approved

**Pull request merged**